src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -42,7 +42,7 @@ let timersyncend = 0;
  * @return 409 if the project path or name are already in use
  * @return 500 if there was an error
  */
-router.post('/api/v1/projects/bind/start', async function (req, res) {await bindStart(req, res)});
+router.post('/api/v1/projects/bind/start', validateReq, async function (req, res) {await bindStart(req, res)});
```

Comment:
Can we change this to be `router.post('/api/v1/projects/bind/start', validateReq, bindStart);` ?